### PR TITLE
Update Windows Astro CLI Installation Instructions

### PR DESCRIPTION
**Pull request by kwcanuck**
Changes initiated by [this Slack conversation](https://astronomer.slack.com/archives/CL44PA4DB/p1675453873470989).

### CODE REVIEW

**Review by jwitz — APPROVED** [review]
LGTM. Do we also need to update the Winget installation per your Slack comments?

> The only issue I encountered when installing the CLI (with winget) was that I needed to add an environment variable to get it to work. 

> I confirmed that astro dev start starts the compose stack and allows you to view through the web browser. I had to update my port assignments...

**Comment by kwcanuck** [comment]
@jwitz that's a very good question! I've been discussing it some SMEs and the consensus seems to be that users shouldn't encounter the same issue that I did. However, to be safe, I'm going to link to a document that provides a solution. Thanks for the reminder.

File: astro/cli/install-cli.md [hunk 1]
```diff
@@ -82,8 +82,6 @@ The winget command line tool is supported on Windows 10 1709 (build 16299) or la
 #### Prerequisites
 
 - [Docker Desktop](https://docs.docker.com/desktop/windows/install/) for Windows.
-- [Docker Engine](https://docs.docker.com/engine/install/) (v1.13.1 or later).
-- [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) enabled on your local machine.
 - Astro CLI version 1.6 or later.
```
Comment (paolaperaza) [inline astro/cli/install-cli.md:85]:
```suggestion
```

Should this be a note before or after the "install a specific CLI version" section? Seems weird to have CLI 1.6 as a prerequisite for installing

Comment (paolaperaza) [reply astro/cli/install-cli.md:85]: Per convo in check-in, let's just remove dis @kwcanuck

File: astro/cli/install-cli.md [hunk 2]
```diff
@@ -109,8 +111,6 @@ This is where you'll find information about installing the Astro CLI on a Window
 #### Prerequisites
 
 - [Docker Desktop](https://docs.docker.com/desktop/windows/install/) for Windows.
```
Comment (rob-1126) [inline astro/cli/install-cli.md:113]:
```suggestion
- One of the following:
* [Docker Desktop](https://docs.docker.com/desktop/windows/install/) in WSL2 mode in conjunction with the linux version of astro-cli.
* [Docker Community-Edition Binaries](https://docs.docker.com/engine/install/binaries/) in WSL2 mode in conjunction with the linux version of astro-cli.
* [Docker Desktop](https://docs.docker.com/desktop/windows/install/) in HyperV mode in conjunction with astro.exe.

```

Comment (kwcanuck) [reply astro/cli/install-cli.md:113]: After much discussion with @rob-1126 this was determined to be the best option.

**Pull request merged**